src/main/java/uk/gov/pay/api/model/CardDetails.java

```diff
@@ -16,21 +18,22 @@
 
     @JsonProperty("first_digits_card_number")
     private final String firstDigitsCardNumber;
-    
+
     @JsonProperty("cardholder_name")
     private final String cardHolderName;
 
     @JsonProperty("expiry_date")
     private final String expiryDate;
 
     @JsonProperty("billing_address")
+    @JsonInclude(value = JsonInclude.Include.NON_EMPTY)
```

Comment:
So now I’m seeing the code, I think the story may have have gotten the requirements wrong here. I think we probably want to include the `"billing_address"` but have it be explicitly `null` to match the behaviour of the other properties inside `"card_details"` e.g. `"first_digits_card_number"` (which won’t be populated for older payments).